databind/src/main/java/gov/nist/secauto/metaschema/databind/io/json/MetaschemaJsonWriter.java

```diff
@@ -128,8 +127,13 @@ public void writeItemFieldValue(Object parentItem, IBoundFieldValue fieldValue)
 
       String valueKeyName;
       if (jsonValueKey != null) {
+        Object keyValue = jsonValueKey.getValue(parentItem);
+        if (keyValue == null) {
+          throw new IOException(String.format("Null JSON value key value for definition '%s'",
```

Comment:
Minor, non-blocking: given the above similar feedback maybe?

```suggestion
          throw new IOException(String.format("Null value for json-value-key for definition '%s'",
```